templates/partials/pageNav-item.hbs

```diff
@@ -7,7 +7,7 @@
   data-id="{{_id}}"
   data-item-index="{{_index}}"
   {{#any _isHidden locked}} disabled="disabled"{{/any}}
-  aria-label="{{{compile ariaLabel this}}} {{#if locked}}{{@root/_globals._accessibility._ariaLabels.locked}}{{else if _isComplete}}{{@root/_globals._accessibility._ariaLabels.complete}}{{else}}{{@root/_globals._accessibility._ariaLabels.incomplete}}{{/if}}"
+  aria-label="{{{compile ariaLabel this}}}; {{#if locked}}{{@root/_globals._accessibility._ariaLabels.locked}}{{/if}}"
```

Comment:
Please put the status at the front witha period as before.

Reply:
Great call. Done.